src/learning-catalogue/model/factory/audienceFactory.ts

```diff
@@ -0,0 +1,23 @@
+import {Audience} from '../audience'
+import * as moment from 'moment'
```

Comment:
This import is different to how you imported moment above - what's the difference? Can we be consistent?

Reply:
Changed.